lib/edamam_api_wrapper.rb

```diff
@@ -0,0 +1,43 @@
+require 'httparty'
+
+class EdamamApiWrapper
+
+  BASE_URL = 'https://api.edamam.com/search'
+  TOKEN = ENV['EDAMAM_TOKEN']
+  ID = ENV['EDAMAM_ID']
+
+  def self.list_recipes(search_terms)
+    url = "#{BASE_URL}?&app_id=#{ID}&app_key=#{TOKEN}&to=50&q=#{search_terms}"
+    data = HTTParty.get(url)
+    recipe_list = []
+    if data["hits"]
+      data["hits"].each do |recipe_data|
+        recipe_list << create_recipe(recipe_data["recipe"])
+      end
+    end
+    return recipe_list
+  end
+
+  def self.list_recipe(id)
+    url = "#{BASE_URL}?&app_id=#{ID}&app_key=#{TOKEN}&r=http://www.edamam.com/ontologies/edamam.owl%23recipe_#{id}"
+    # Send the request
+    HTTParty.get(url).parsed_response
```

Comment:
You should have some code in here to return `nil` or something if the `id` doesn't work.